portafly/doc/ux_specs/backend/create_backend.md

```diff
@@ -0,0 +1,44 @@
+# Create Backend
```

Comment:
@deernano The PR is about the Backend Overview page, so this file shouldn't be part of it. To avoid conflicts it would be better to delete this file from the PR.

When creating a new branch for writing the UX specs for a new page, it would be better to always create the new branch from `master`. In this way, you will only submit the new changes and not all the previous ones too :)

Reply:
I c, I ll delete it